Лабораторная работа 1/Лаба 1(Замена пропущенного элементасредним арифметическим.py

```diff
@@ -0,0 +1,26 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+
+summa = 0
+count = 0
+
+for i in range(len(numbers)):
+    if type(numbers[i]) == int:
+        summa += numbers[i]
+        count += 1
+
+average = summa / len(numbers)
+
+
+for i in range(len(numbers)):
+    if numbers[i] is None:
+        numbers[i] = average
```

Comment:
Интересный метод решения с использованием циклов! 
Предлагаю вам альтернативный вариант решения, которой строится на методах, а в последующем, узнав о сложности выполнения тех или иных операций в коде, еще и будет менее ресурсоемким, так как не требует перебора всех значений в списке: 
```suggestion
numbers[numbers.index(None)] = (sum(numbers[:numbers.index(None)]) + sum(numbers[numbers.index(None) + 1:])) / (len(numbers))
```